packages/contracts/src/automation/example/SimpleGelatoJob.sol

```diff
@@ -7,17 +7,25 @@ import {SafeUUPSUpgradeable} from "../../upgradeable/SafeUUPSUpgradeable.sol";
 import {SafeInitializable} from "../../upgradeable/SafeInitializable.sol";
 import {IVersionable} from "../../upgradeable/IVersionable.sol";
 
+/// Example of a simple Gelato job implementation
 contract SimpleGelatoJob is GelatoJobAdapter, SafeUUPSUpgradeable {
     uint256 public workMethodCalledCounter;
     bool public canWorkResult = false;
 
     /// @inheritdoc IVersionable
     function version() external pure override returns (string memory) {
-        return "0.1.1";
+        return "0.1.2";
     }
 
     // allow sending eth to the test contract
     receive() external payable {} // solhint-disable-line no-empty-blocks
+    
+    /// @dev not need in real job
+    /// This method added only because the OpenZepplin Defender marks it as a security issue
+    function withdraw(uint256 amount) public onlyOwner {
+        require(address(this).balance >= amount, "Insufficient balance in the contract");
```

Comment:
why not separate error? it's more efficient

Reply:
...efficient in case of gas usage on deployment

Reply:
It is just a fix for the defender, for a contract that is kept only as an example. I wanted to keep it short